examples/demo_threading/cpp/helpers/sensor.h

```diff
@@ -49,6 +49,6 @@ private Q_SLOTS:
 
 private:
   ::std::optional<int> m_index;
-  EnergyUsageProxyModel* m_model;
+  EnergyUsageProxyModel* m_model = nullptr;
```

Comment:
Can you do this in the constructor implementation instead, to keep consistency with elsewhere.

```cpp
Sensor::Sensor(QObject* parent)
   : QObject(parent)
   , m_model(nullptr)
{
...
}
```

Reply:
Ok I will change it.